examples/resources/awscc_apigatewayv2_route/apigatewayv2_route.tf

```
@@ -0,0 +1,18 @@
+resource "awscc_apigatewayv2_api" "example_api" {
+  name                       = "example-websocket-api"
+  protocol_type              = "WEBSOCKET"
+  route_selection_expression = "$request.body.action"
+  tags = [{
+    key   = "Modified By"
+    value = "AWSCC"
+  }]
+}
+
+resource "awscc_apigatewayv2_route" "example_route" {
+  api_id    = awscc_apigatewayv2_api.example_api.id
+  route_key = "$default"
```

```suggestion
  route_key = "default"
```
unless there is specific need for using `$default` in the route path, I suggest we avoid special char

---

For Websockets on API Gateways  `$default` is one of the predefined routes along with `$connect` and `$disconnect`, for that reason I believe it's appropriate to use `$default` to remain consistent with docs

https://docs.aws.amazon.com/apigateway/latest/developerguide/websocket-api-develop-routes.html